data.sql

```diff
@@ -3,4 +3,11 @@
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (1, 'Agumon', '03-02-2020',0, true, 10.23);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (2, 'Gabumon', '2018-11-15',2, true, 8.00);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (3, 'Pikachu', '07-01-2021',1, false, 15.04);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (4, 'Agumon', '12-05-2017',5, true, 10.23);
\ No newline at end of file
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  overriding system value values (4, 'Devimon', '12-05-2017',5, true, 10.23);
```

Comment:
- [ ] OPTIONAL: I noticed that you are adding primary key explicitly everywhere, I would recommend to set to AUTO_INCREMENT and then you don't have to increment it explicitly every time 😉 
```
CREATE TABLE Persons (
    Personid int NOT NULL AUTO_INCREMENT,
    LastName varchar(255) NOT NULL,
    FirstName varchar(255),
    Age int,
    PRIMARY KEY (Personid)
);
```